0x/Utilities/EthereumAddress.cs

```diff
@@ -57,5 +61,13 @@ public override string ToString()
                 throw new InvalidCastException($"Can not cast string to EthereumAddress", ex);
```

Comment:
`InvalidCastException` or `ArgumentNullException`?

Reply:
This is constructor so ArgumentNullException is suitable.